src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -606,7 +606,36 @@ def load_preterm_prevalence(
         draw_cols
     ].sum()
 
-    return sum_exposure
+    enn_data = sum_exposure.reset_index()
+    enn_data["child_age_start"] = data_values.EARLY_NEONATAL_AGE_START
+    enn_data["child_age_end"] = data_values.LATE_NEONATAL_AGE_START
+
+    # get late neonatal prevalence from our PAF simulation
+    filename = "calculated_late_neonatal_preterm_prevalence.parquet"
+    location_mapper = {
+        "Ethiopia": "ethiopia",
+        "Nigeria": "nigeria",
+        "Pakistan": "pakistan",
+    }
```

Comment:
Is this necessary? Why not just `location.lower()`?

Reply:
I copy pasted this from somewhere else and thought the same but decided not to change it for consistency. Now that you've suggested it though, I'm OK with switching to lower() in both places.